assets/app/view/game/part/revenue.rb

```diff
@@ -31,12 +31,12 @@ def preferred_render_locations
               {
                 region_weights: TOP_ROW,
                 x: 0,
-                y: -48,
+                y: -72,
```

Comment:
does this change the region_weights?

Reply:
No. I originally thought it was a problem with the weights, but the actual placement needed to be lined up with what the weights were saying.

Reply:
No. Just the position.

Reply:
But my latest commit does change the region_weights: reduced number of locations (to avoid having revenue appear at the very top or bottom of a tile) and a greater than unity weight (to bias location text from appearing on top of the revenue).